streamhist/histogram.py

```diff
@@ -81,6 +81,11 @@ def update(self, n, count=1):
             for p in n:
                 self.update(p, count)  # Count is assumed to apply for all
         else:
+            # NaN is not equal to itself
+            if n != n:
```

Comment:
can't you do something like this https://stackoverflow.com/questions/944700/how-can-i-check-for-nan-values?

Reply:
so math.isnan throws type errors as it accepts only real numbers. I guess we can use math.isnan as we don’t get any string inputs right now. But if we try to generate histograms for categorical columns, then we'll need to change this.